tests/unit/data_handling_test.py

```diff
@@ -4,23 +4,28 @@
 
 def test_batches():
     device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
-    dataset = VisXPData(
-        datapath="data/input-files/test_source_id",
-        model_config_file="model/model_config.yml",
-        device=device,
-        check_spec_dim=False,
-    )
-    for i, item in enumerate(dataset.batches(1)):
-        index = int(item["timestamp"][0])
+    for audio_too in [True, False]:
```

Comment:
instead of looping through True, False for audio_too within the test, you could consider using a pytest parameter to run the test for each case, it's a little more readable that way.